src/outer_sum.jl

```diff
@@ -0,0 +1,29 @@
+function outer_sum(xx, yy)
```

Comment:
this function should not be needed thanks to Julia broadcast.
if x and y are vectors, then `x + y'` will be the same as `outer_sum(x,y)`